diff -puNrb tcpdump-3.8.3/addrtoname.c tcpdump/addrtoname.c
--- tcpdump-3.8.3/addrtoname.c	2011-06-15 11:24:54.000000000 +0300
+++ tcpdump/addrtoname.c	2021-12-03 14:02:44.416539333 +0200
@@ -677,6 +677,7 @@ ipxsap_string(u_short port)
 static void
 init_servarray(void)
 {
+#if 0
 	struct servent *sv;
 	register struct hnamemem *table;
 	register int i;
@@ -703,6 +704,7 @@ init_servarray(void)
 		table->nxt = newhnamemem();
 	}
 	endservent();
+#endif
 }
 
 /*XXX from libbpfc.a */
diff -puNrb tcpdump-3.8.3/config.sub tcpdump/config.sub
--- tcpdump-3.8.3/config.sub	2011-06-15 11:24:54.000000000 +0300
+++ tcpdump/config.sub	2021-12-03 14:02:44.416539333 +0200
@@ -3,7 +3,7 @@
 #   Copyright (C) 1992, 1993, 1994, 1995, 1996, 1997, 1998, 1999,
 #   2000, 2001, 2002, 2003 Free Software Foundation, Inc.
 
-timestamp='2003-11-03'
+timestamp='2021-07-19'
 
 # This file is (in principle) common to ALL GNU software.
 # The presence of a machine in this file suggests that SOME GNU software
@@ -1133,7 +1133,7 @@ case $os in
 	# The portable systems comes first.
 	# Each alternative MUST END IN A *, to match a version number.
 	# -sysv* is not here because it comes later, after sysvr4.
-	-gnu* | -bsd* | -mach* | -minix* | -genix* | -ultrix* | -irix* \
+	-gnu* | -bsd* | -mach* | -minix* | -genix* | -ultrix* | -irix* | -musl* \
 	      | -*vms* | -sco* | -esix* | -isc* | -aix* | -sunos | -sunos[34]*\
 	      | -hpux* | -unos* | -osf* | -luna* | -dgux* | -solaris* | -sym* \
 	      | -amigaos* | -amigados* | -msdos* | -newsos* | -unicos* | -aof* \
diff -puNrb tcpdump-3.8.3/configure tcpdump/configure
--- tcpdump-3.8.3/configure	2011-06-15 11:24:54.000000000 +0300
+++ tcpdump/configure	2021-12-02 11:14:07.313513112 +0200
@@ -3384,7 +3384,7 @@ echo $ECHO_N "checking Linux kernel vers
  		if test "${ac_cv_linux_vers+set}" = set; then
   echo $ECHO_N "(cached) $ECHO_C" >&6
 else
-  ac_cv_linux_vers=unknown
+  ac_cv_linux_vers=2.4.21
 fi
 
  	else
@@ -9730,8 +9730,7 @@ done
 echo "$as_me:$LINENO: result: $ac_cv_ssleay_path" >&5
 echo "${ECHO_T}$ac_cv_ssleay_path" >&6
 if test "$ac_cv_ssleay_path" != no; then
-	V_INCLS="$V_INCLS $incdir"
-	LDFLAGS="-L$dir/lib $LDFLAGS"
+	LDFLAGS="$LDFLAGS"
 	if test -f $ac_cv_ssleay_path/lib/libRSAglue.a; then
 		LIBS="$LIBS -lRSAglue"
 	fi
diff -puNrb tcpdump-3.8.3/Makefile.in tcpdump/Makefile.in
--- tcpdump-3.8.3/Makefile.in	2011-06-15 11:24:54.000000000 +0300
+++ tcpdump/Makefile.in	2021-12-02 11:14:07.309513101 +0200
@@ -77,11 +77,11 @@ CSRC =	addrtoname.c gmpls.c gmt2local.c
 	print-ipx.c print-isakmp.c print-isoclns.c print-krb.c \
 	print-l2tp.c print-lane.c print-ldp.c print-llc.c \
 	print-lwres.c print-mobile.c print-mpls.c print-msdp.c \
-	print-nfs.c print-ntp.c print-null.c print-ospf.c \
+	print-ntp.c print-null.c print-ospf.c \
 	print-pflog.c print-pim.c print-ppp.c print-pppoe.c \
 	print-pptp.c print-radius.c print-raw.c print-rip.c \
 	print-rsvp.c print-rx.c print-sctp.c print-sl.c print-sll.c \
-	print-snmp.c print-stp.c print-sunatm.c print-sunrpc.c \
+	print-snmp.c print-stp.c print-sunatm.c \
 	print-tcp.c print-telnet.c print-tftp.c print-timed.c \
 	print-token.c print-udp.c print-vjc.c print-vrrp.c \
 	print-wb.c print-zephyr.c setsignal.c tcpdump.c util.c
diff -puNrb tcpdump-3.8.3/print-sunrpc.c tcpdump/print-sunrpc.c
--- tcpdump-3.8.3/print-sunrpc.c	2011-06-15 11:24:54.000000000 +0300
+++ tcpdump/print-sunrpc.c	2021-12-02 11:14:07.321513134 +0200
@@ -141,12 +141,12 @@ progstr(prog)
 
 	if (lastprog != 0 && prog == lastprog)
 		return (buf);
-#ifndef WIN32
+#if 0
 	rp = getrpcbynumber(prog);
 	if (rp == NULL)
-#endif /* WIN32 */
+#endif
 		(void) snprintf(buf, sizeof(buf), "#%u", prog);
-#ifndef WIN32
+#if 0
 	else
 		strlcpy(buf, rp->r_name, sizeof(buf));
 #endif
diff -puNrb tcpdump-3.8.3/print-tcp.c tcpdump/print-tcp.c
--- tcpdump-3.8.3/print-tcp.c	2011-06-15 11:24:54.000000000 +0300
+++ tcpdump/print-tcp.c	2021-12-02 11:14:07.321513134 +0200
@@ -30,7 +30,9 @@ static const char rcsid[] _U_ =
 
 #include <tcpdump-stdinc.h>
 
+#if 0
 #include <rpc/rpc.h>
+#endif
 
 #include <stdio.h>
 #include <stdlib.h>
@@ -211,6 +213,7 @@ tcp_print(register const u_char *bp, reg
 
 	hlen = TH_OFF(tp) * 4;
 
+#if 0
 	/*
 	 * If data present and NFS port used, assume NFS.
 	 * Pass offset of data plus 4 bytes for RPC TCP msg length
@@ -230,6 +233,8 @@ tcp_print(register const u_char *bp, reg
 			return;
 		}
 	}
+#endif
+
 #ifdef INET6
 	if (ip6) {
 		if (ip6->ip6_nxt == IPPROTO_TCP) {
diff -puNrb tcpdump-3.8.3/print-udp.c tcpdump/print-udp.c
--- tcpdump-3.8.3/print-udp.c	2011-06-15 11:24:54.000000000 +0300
+++ tcpdump/print-udp.c	2021-12-02 11:14:07.325513145 +0200
@@ -35,7 +35,11 @@ static const char rcsid[] _U_ =
 #endif
 #include <arpa/tftp.h>
 
+#if 0
 #include <rpc/rpc.h>
+#endif
+
+#define FALSE 0
 
 #include <stdio.h>
 #include <string.h>
@@ -468,8 +472,10 @@ udp_print(register const u_char *bp, u_i
 		return;
 	}
 	if (packettype) {
+#if 0
 		register struct rpc_msg *rp;
 		enum msg_type direction;
+#endif
 
 		switch (packettype) {
 
@@ -483,6 +489,7 @@ udp_print(register const u_char *bp, u_i
 			wb_print((void *)(up + 1), length);
 			break;
 
+#if 0
 		case PT_RPC:
 			rp = (struct rpc_msg *)(up + 1);
 			direction = (enum msg_type)EXTRACT_32BITS(&rp->rm_direction);
@@ -493,6 +500,7 @@ udp_print(register const u_char *bp, u_i
 				nfsreply_print((u_char *)rp, length,
 				    (u_char *)ip);			/*XXX*/
 			break;
+#endif
 
 		case PT_RTP:
 			udpipaddr_print(ip, sport, dport);
@@ -533,6 +541,7 @@ udp_print(register const u_char *bp, u_i
 		return;
 	}
 
+#if 0
 	if (!qflag) {
 		register struct rpc_msg *rp;
 		enum msg_type direction;
@@ -566,6 +575,7 @@ udp_print(register const u_char *bp, u_i
 			return;
 		}
 	}
+#endif
 	udpipaddr_print(ip, sport, dport);
 
 	if (IP_V(ip) == 4 && vflag && !fragmented) {
