diff -puNrb e2fsprogs-1.46.2/config/config.sub e2fsprogs/config/config.sub
--- e2fsprogs-1.46.2/config/config.sub	2021-03-15 14:20:09.000000000 +0200
+++ e2fsprogs/config/config.sub	2021-12-03 14:02:42.948535474 +0200
@@ -1338,7 +1338,7 @@ case $os in
 	# The portable systems comes first.
 	# Each alternative MUST end in a * to match a version number.
 	# sysv* is not here because it comes later, after sysvr4.
-	gnu* | bsd* | mach* | minix* | genix* | ultrix* | irix* \
+	gnu* | bsd* | mach* | minix* | genix* | ultrix* | irix* | musl* \
 	     | *vms* | esix* | aix* | cnk* | sunos | sunos[34]*\
 	     | hpux* | unos* | osf* | luna* | dgux* | auroraux* | solaris* \
 	     | sym* | kopensolaris* | plan9* \
diff -puNrb e2fsprogs-1.46.2/e2fsck/logfile.c e2fsprogs/e2fsck/logfile.c
--- e2fsprogs-1.46.2/e2fsck/logfile.c	2021-03-15 14:20:09.000000000 +0200
+++ e2fsprogs/e2fsck/logfile.c	2021-12-03 14:02:42.964535515 +0200
@@ -77,7 +77,7 @@ static void expand_percent_expression(e2
 	if ((ch == 'D') || (ch == 'd') || (ch == 'm') || (ch == 'y') ||
 	    (ch == 'Y') ||
 	    (ch == 'T') || (ch == 'H') || (ch == 'M') || (ch == 'S')) {
-		tzset();
+//		tzset();
 		tm = (*flags & FLAG_UTC) ? gmtime_r(&ctx->now, &tm_struct) :
 			localtime_r(&ctx->now, &tm_struct);
 	}
diff -puNrb e2fsprogs-1.46.2/e2fsck/unix.c e2fsprogs/e2fsck/unix.c
--- e2fsprogs-1.46.2/e2fsck/unix.c	2021-03-15 14:20:09.000000000 +0200
+++ e2fsprogs/e2fsck/unix.c	2021-12-03 14:02:42.968535525 +0200
@@ -241,6 +241,8 @@ static void check_mount(e2fsck_t ctx)
 			ctx->filesystem_name);
 		return;
 	}
+	if (ctx->mount_flags & EXT2_MF_READONLY)
+	    return; /* disable root fs filesystem check */
 
 	/*
 	 * If the filesystem isn't mounted, or it's the root
@@ -1499,8 +1501,9 @@ restart:
 	if ((ctx->options & E2F_OPT_READONLY) == 0) {
 		flags |= EXT2_FLAG_RW;
 		if (!(ctx->mount_flags & EXT2_MF_ISROOT &&
-		      ctx->mount_flags & EXT2_MF_READONLY))
-			flags |= EXT2_FLAG_EXCLUSIVE;
+		      ctx->mount_flags & EXT2_MF_READONLY)) {
+			//flags |= EXT2_FLAG_EXCLUSIVE;
+		}
 		if ((ctx->mount_flags & EXT2_MF_READONLY) &&
 		    (ctx->options & E2F_OPT_FORCE))
 			flags &= ~EXT2_FLAG_EXCLUSIVE;
diff -puNrb e2fsprogs-1.46.2/lib/blkid/llseek.c e2fsprogs/lib/blkid/llseek.c
--- e2fsprogs-1.46.2/lib/blkid/llseek.c	2021-03-15 14:20:09.000000000 +0200
+++ e2fsprogs/lib/blkid/llseek.c	2021-12-03 14:02:42.972535536 +0200
@@ -56,7 +56,7 @@ extern long long llseek(int fd, long lon
 
 #else /* SIZEOF_LONG != SIZEOF_LONG_LONG */
 
-#include <linux/unistd.h>
+//#include <linux/unistd.h>
 
 #ifndef __NR__llseek
 #define __NR__llseek            140
diff -puNrb e2fsprogs-1.46.2/lib/ext2fs/llseek.c e2fsprogs/lib/ext2fs/llseek.c
--- e2fsprogs-1.46.2/lib/ext2fs/llseek.c	2021-03-15 14:20:09.000000000 +0200
+++ e2fsprogs/lib/ext2fs/llseek.c	2021-12-03 14:02:42.988535578 +0200
@@ -57,7 +57,7 @@ extern long long llseek (int fd, long lo
 
 #else /* SIZEOF_LONG != SIZEOF_LONG_LONG */
 
-#include <linux/unistd.h>
+//#include <linux/unistd.h>
 
 #ifndef __NR__llseek
 #define __NR__llseek            140
