diff -puNrb libpcap-0.8.3/config.sub libpcap/config.sub
--- libpcap-0.8.3/config.sub	2011-06-15 11:08:09.000000000 +0300
+++ libpcap/config.sub	2021-12-03 14:02:43.612537219 +0200
@@ -3,7 +3,7 @@
 #   Copyright (C) 1992, 1993, 1994, 1995, 1996, 1997, 1998, 1999,
 #   2000, 2001, 2002, 2003 Free Software Foundation, Inc.
 
-timestamp='2003-11-03'
+timestamp='2021-07-19'
 
 # This file is (in principle) common to ALL GNU software.
 # The presence of a machine in this file suggests that SOME GNU software
@@ -1133,7 +1133,7 @@ case $os in
 	# The portable systems comes first.
 	# Each alternative MUST END IN A *, to match a version number.
 	# -sysv* is not here because it comes later, after sysvr4.
-	-gnu* | -bsd* | -mach* | -minix* | -genix* | -ultrix* | -irix* \
+	-gnu* | -bsd* | -mach* | -minix* | -genix* | -ultrix* | -irix* | -musl* \
 	      | -*vms* | -sco* | -esix* | -isc* | -aix* | -sunos | -sunos[34]*\
 	      | -hpux* | -unos* | -osf* | -luna* | -dgux* | -solaris* | -sym* \
 	      | -amigaos* | -amigados* | -msdos* | -newsos* | -unicos* | -aof* \
diff -puNrb libpcap-0.8.3/configure libpcap/configure
--- libpcap-0.8.3/configure	2011-06-15 11:08:09.000000000 +0300
+++ libpcap/configure	2021-12-02 11:14:05.905509301 +0200
@@ -4253,7 +4253,7 @@ echo $ECHO_N "checking Linux kernel vers
  		if test "${ac_cv_linux_vers+set}" = set; then
   echo $ECHO_N "(cached) $ECHO_C" >&6
 else
-  ac_cv_linux_vers=unknown
+  ac_cv_linux_vers=2.4.21
 fi
 
  	else
diff -puNrb libpcap-0.8.3/Makefile.in libpcap/Makefile.in
--- libpcap-0.8.3/Makefile.in	2011-06-15 11:08:09.000000000 +0300
+++ libpcap/Makefile.in	2021-12-02 11:14:05.897509280 +0200
@@ -100,7 +100,7 @@ all: libpcap.a
 
 libpcap.a: $(OBJ)
 	@rm -f $@
-	ar rc $@ $(OBJ) $(LIBS)
+	$(AR) rc $@ $(OBJ) $(LIBS)
 	$(RANLIB) $@
 
 scanner.c: $(srcdir)/scanner.l
diff -puNrb libpcap-0.8.3/pcap-linux.c libpcap/pcap-linux.c
--- libpcap-0.8.3/pcap-linux.c	2011-06-15 11:08:09.000000000 +0300
+++ libpcap/pcap-linux.c	2021-12-03 14:02:43.612537219 +0200
@@ -145,10 +145,6 @@ static const char rcsid[] _U_ =
 #include <linux/filter.h>
 #endif
 
-#ifndef __GLIBC__
-typedef int		socklen_t;
-#endif
-
 #ifndef MSG_TRUNC
 /*
  * This is being compiled on a system that lacks MSG_TRUNC; define it
